render/render.js

```diff
@@ -851,7 +851,13 @@ module.exports = function($window) {
 		var result
 		if (typeof handler === "function") result = handler.call(ev.currentTarget, ev)
 		else if (typeof handler.handleEvent === "function") handler.handleEvent(ev)
-		if (this._ && ev.redraw !== false) (0, this._)()
+		if (this._ && ev.redraw !== false) {
+			(0, this._)()
+			if (result != null) {
+				if (typeof result.finally === "function") result.finally((0, this._))
```

Comment:
Let's just use `.then`. It's redundant to do both that and `.finally`, because per spec `p.finally(f)` is roughly just `.then(async v => { await f(); return v }, async e => { await f(); throw e })`, complete with the `.then` method call being observable.